docs/guides/getting-started/setup/next-js.mdx

```diff
@@ -56,10 +56,10 @@ pnpm create next-app --use-pnpm --typescript
   </TabItem>
 </Tabs>
 
-1. _Project name_  
+1. _Project name_
```

Comment:
The spaces after the headings are intentional so that the line below is indeed a newline (see the deploy preview for this page.